applications/CompressiblePotentialFlowApplication/tests/naca0012_small_adjoint_test/cl_jump_ref.dat

```diff
@@ -0,0 +1 @@
+0.3230253050805644
```

Comment:
One file to store a double? Please find another way to check this.

Reply:
This is solved in https://github.com/KratosMultiphysics/Kratos/pull/4668